src/copy_member_to_members.py

```diff
@@ -3,8 +3,8 @@
 """
 from os import environ
 
-import mappings
-from service import build_es_client
+from src import mappings
```

Comment:
Maybe these imports can be improved. If you have a suggestion, please let me know.

Reply:
That should be fine, but you know more Python than I do right now. One question, did you mean to remove this line?
```python
from service import build_es_client
```

Reply:
No, I meant not to use two lines when importing for the same package

-` from src import this`
- `from src import that`
I will squeeze the imports in one import.